src/components/Gallery/index.test.tsx

```diff
@@ -321,7 +321,7 @@ describe('the gallery block', () => {
       EventEmitter.subscribe('galleryAutoplayStart', (event: GalleryEventData) => eventHandler(event, 1));
       EventEmitter.subscribe('galleryAutoplayStop', (event: GalleryEventData) => eventHandler(event, 2));
       autoBtnWrapper.simulate('click');
-      sleep(5000).then(() => {
+      sleep(10000).then(() => {
```

Comment:
was this intentional? I'd prefer a better way of doing this if so @npak243. Why was this needing to be changed?

Reply:
`sleep(5000).then(() => {` works for me locally and in the cloud when the test is run. Can you push this up without the change to the sleep amount.

Reply:
@JackHowa Yes, I will revert this change, thanks for your review and new PR so much.

Reply:
I have reverted this commit, please help to review it again. Thank for your nice help a lot.

Reply:
thanks, will take another look